examples/integration_test.py

```diff
@@ -402,6 +402,32 @@ def print_commit_result(err, partitions):
         print('# Committed offsets for: %s' % partitions)
 
 
+def verify_consumer_seek(c, seek_to_msg):
+    """ Seek to message and verify the next consumed message matches.
+        Must only be performed on an actively consuming consumer. """
+
+    tp = confluent_kafka.TopicPartition(seek_to_msg.topic(),
+                                        seek_to_msg.partition(),
+                                        seek_to_msg.offset())
+    print('seek: Seeking to %s' % tp)
+    c.seek(tp)
+
+    while True:
+        msg = c.poll()
+        assert msg is not None
+        if msg.error():
+            print('seek: Ignoring non-message: %s' % msg)
+            continue
+
+        if msg.topic() != seek_to_msg.topic() or msg.partition() != seek_to_msg.partition():
```

Comment:
i think this would be better as an assert. in practice, the test'll still fail, but i think more correct as an assert.

Reply:
The topic may have multiple partitions and the subscribe() will most likely have all partitions assigned, but we only care about one single partition here, the one we called seek() on.
So it should just ignore messages for other partitions, an assert would not be correct.